### PR TITLE
Update Makefile with new ranger version

### PR DESCRIPTION
**Pull request by junderw**

### CODE REVIEW

File: src/statistics.hpp [hunk 1]
```diff
@@ -25,7 +25,7 @@ struct dumpOutputValuesOverHeight : public TransformBase<Block> {
 				fwrite(buffer.begin(), buffer.size(), 1, stdout);
 			}
 
-			transactions.pop_front();
```
Comment (dcousens) [inline src/statistics.hpp]: `pop_front` usage is OK here,  the new `ranger` doesn't support `pop_front(N)`

**Comment by junderw** [comment]
@dcousens looking ok? If everything's good I'll squash and merge

File: Makefile [hunk 2]
```diff
@@ -32,10 +32,10 @@ include/hexxer.hpp:
 	curl 'https://raw.githubusercontent.com/dcousens/hexxer/d76f9e526676535fd4c2584a8f84582994c35996/hexxer.h' > $@
 
 include/ranger.hpp:
-	curl 'https://raw.githubusercontent.com/dcousens/ranger/e5b156dd0d8d3de14c2c807a2d7874af0e9cd3e3/ranger.hpp' > $@
+	curl 'https://raw.githubusercontent.com/dcousens/ranger/ddf75d976cb2e1bd4a5b69543b36f0778f90762b/ranger.hpp' > $@
 
 include/serial.hpp:
-	curl 'https://raw.githubusercontent.com/dcousens/ranger/e5b156dd0d8d3de14c2c807a2d7874af0e9cd3e3/serial.hpp' > $@
+	curl 'https://raw.githubusercontent.com/dcousens/ranger/ddf75d976cb2e1bd4a5b69543b36f0778f90762b/serial.hpp' > $@
```
Comment (dcousens) [inline Makefile]: 44d9037b29a91be0dfce178a8a67314530919e45 is latest HEAD

Comment (junderw) [reply Makefile]: oops. wonder where I copied that commit from... it's nowhere in ranger...

**Comment by junderw** [comment]
@dcousens done

**Pull request merged**